.gitignore

```diff
@@ -34,3 +34,5 @@ build
 
 # STAC schema cache
 stac_schema_cache.sqlite
+
+_version.py
```

Comment:
I'm guessing you want to ignore the file generated by setuptools-scm, but won't this ignore a file called `_version.py` anywhere in the repository?

Reply:
Yep, hopefully we don't have any other files with that name.

I'll make it a more specific exclude.